integration/it-integration/src/main/java/se/inera/intyg/rehabstod/integration/it/dto/PopulateFiltersRequestDTO.java

```diff
@@ -19,42 +19,13 @@
 
 package se.inera.intyg.rehabstod.integration.it.dto;
 
+import lombok.Data;
+
+@Data
 public class PopulateFiltersRequestDTO {
     private String unitId;
     private String careUnitId;
+    private String doctorId;
     private int maxCertificateGap;
```

Comment:
maxCertificateGap is not used in our PopulateFiltersRequest, atleast not yet. Maybe we should remove it for now and add it later on if we actually need it?